projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -2427,6 +2427,11 @@ public function internal_personal_data_formater( $post_ids ) {
 				'value' => $feedback->get_ip_address(),
 			);
 
+			$post_export_data[] = array(
+				'name'  => __( 'Country Code', 'jetpack-forms' ),
```

Comment:
```suggestion
				'name'  => __( 'Country code', 'jetpack-forms' ),
```

Sentence case everywhere always ;-)